main.go

```diff
@@ -438,6 +438,12 @@ func init() {
 		return []string{"1"}, cobra.ShellCompDirectiveNoFileComp
 	})
 
+	rootCmd.PersistentFlags().IntP("vertical-header", "V", 0, "number of vertical header lines to be displayed constantly")
+	_ = viper.BindPFlag("general.VerticalHeader", rootCmd.PersistentFlags().Lookup("vertical-header"))
+	_ = rootCmd.RegisterFlagCompletionFunc("vertical-header", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		return []string{"1"}, cobra.ShellCompDirectiveNoFileComp
```

Comment:
Could you explain me this pattern?

The field is an integer, and so here you tell viper to complete the field will the value `1`, why 1? Why not []string{}

Is it a way to complete with something that will help user to understand it's a number that is expected?

I had already looked at the code and I was surprised to see this

Here you are doing it again, either on purpose or by copy pasting code.

Reply:
This is still a temporary option, so it doesn't make much sense.
This indicates that an option argument requires numerical values ​​(1 or more).
I thought that the blank ("") wouldn't know what to specify.

Reply:
I'll have to make my own test because it looks strange to me